environments/linux-amd64-wine_osxcross

```diff
@@ -14,26 +14,25 @@ RUN apt-get -y install libjs-mathjax python-yaml lib32gcc1 lib32stdc++6 libc6-i3
 RUN dpkg -i *.deb
 
 RUN git clone https://github.com/tpoechtrager/osxcross /root/osxcross
-
 COPY darwin/SDK/MacOSX10.11.sdk.tar.xz /root/osxcross/tarballs/
-
 RUN cd /root/osxcross && UNATTENDED=1 ./build.sh
 
 ENV PATH="/root/osxcross/target/bin:${PATH}"
 ENV MACOSX_DEPLOYMENT_TARGET="10.11"
+ENV FRAMEWORK="10.11"
+ENV FAUDIO="19.05"
+ENV MOLTENVK="1.1.106.0"
+ENV VKD3D="vkd3d-1.1"
 
 RUN mkdir -p /root/osxcross/target/macports
 RUN printf "packages.macports.org" > /root/osxcross/target/macports/MIRROR
 
-RUN osxcross-macports -universal fakeinstall -v pulseaudio
-
 # We need to compile ncurses by ourselves (see above) to avoid terminals database is inaccessible error
 # This will prevent wine-devel from installing it
-RUN env osxcross-macports -universal fakeinstall ncurses
-
-RUN osxcross-macports -universal install -v wine-devel
-RUN osxcross-macports -universal install gnutls
-# RUN env MACOSX_DEPLOYMENT_TARGET=10.12 osxcross-macports install MoltenVK
+RUN osxcross-macports -universal fakeinstall ncurses
+RUN osxcross-macports -universal fakeinstall pulseaudio
+RUN osxcross-macports -universal install wine-devel
+# RUN osxcross-macports -universal install gnutls
```

Comment:
How will this be replaced?

Reply:
Same as the original plan building it from source, I wanted this to at least build something that worked for the moment, once I'm properly free I can look into doing it again.